apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowKey.cs

```diff
@@ -100,22 +134,47 @@ public override string ToString()
         /// <param name="x"></param>
         /// <param name="y"></param>
         /// <returns></returns>
-        public static bool operator ==(RowKey x, RowKey y)
-        {
-            throw new NotImplementedException();
-        }
+        public static bool operator <(RowKey x, RowKey y) => Compare(x, y) < 0;
 
         /// <summary>
         /// 
         /// </summary>
         /// <param name="x"></param>
         /// <param name="y"></param>
         /// <returns></returns>
-        public static bool operator !=(RowKey x, RowKey y)
-        {
-            throw new NotImplementedException();
-        }
+        public static bool operator <=(RowKey x, RowKey y) => Compare(x, y) <= 0;
 
+        /// <summary>
```

Comment:
We'll probably want to optimize this - in particular, if the keys are of different lengths, there's no point in comparing them as far as they're the same.

Reply:
Done (for ColumnQualifier too)